### ISSUE TITLE
Credly badges for mentors + mentees

### ISSUE THREAD

**Issue by aliok** [cncf/mentoring#1105]
It would be super nice if we can give Credly badges for mentors + mentees across all programs we participate in.

**Comment by nate-double-u** [comment]
I completely agree, and think we can probably target the 1st or second LFX term next year.

**Comment by aliok** [comment]
I think we can even bring this up to Google for GSoC; although I think someone must have already asked them :)

**Comment by aliok** [comment]
@nate-double-u learnings from this exercise can be actually used in other places like CNCF projects. https://github.com/knative/community/issues/1493

**Comment by aliok** [comment]
Related: https://github.com/cncf/toc/discussions/1248
